e2e/cypress/integration/messaging/message_spec.js

```diff
@@ -182,4 +182,22 @@ describe('Message', () => {
         // * Focus to remain in the RHS text box
         cy.get('#reply_textbox').should('be.focused');
     });
+
+    it('MM-T1797 Compact view: Show single image thumbnail', () => {
+        const filename = 'image-small-height.png';
+
+        // Make a post with some text and a single image
+        cy.get('div[id="centerChannelFooter"]').find('input[id="fileUploadInput"]').attachFile(filename);
```

Comment:
Can be simplified to,
```suggestion
        cy.get('#centerChannelFooter').find('#fileUploadInput').attachFile(filename);
```